.travis.yml

```diff
@@ -9,11 +9,19 @@ cache:
     - lib
     - node_modules
 
+install:
+  - yarn add codecov
+
 script:
   - yarn build
   - yarn test --coverage
   - yarn lint
   - bin/run -v
+  - yarn add codecov
```

Comment:
It's not necessary since we install the library with other dependencies.

Reply:
Ok, I'll remove it.